apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -148,14 +148,26 @@ export function InstructorCourseAdminSettings({
                 </div>
                 <div class="form-group">
                   <label for="repository">Repository</label>
-                  <input
-                    type="text"
-                    class="form-control"
-                    id="repository"
-                    name="repository"
-                    value="${resLocals.course.repository}"
-                    disabled
-                  />
+                  <span class="input-group">
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="repository"
+                      name="repository"
+                      value="${resLocals.course.repository}"
+                      disabled
+                    />
+                    <div class="input-group-append">
```

Comment:
Consider making this conditional depending on if `resLocals.course.repository` is truthy. It will almost always be non-null in production but it may be null locally, in which case it's probably best to hide, or at least disable the button.